main.go

```diff
@@ -295,6 +271,45 @@ func main() {
 	}
 }
 
+func setupDrainController(mgr ctrl.Manager, restConfig *rest.Config,
+	platformsHelper platforms.Interface, scheme *runtime.Scheme) error {
+	if vars.UseExternalDrainer {
+		setupLog.Info("'UseExternalDrainer' is set, draining will be done externally")
```

Comment:
please add a note on why we still setup the drain controller here even if UseExternalDrainer is set.